### PR TITLE
Install config files to /usr/local/share/colin/

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Signed-off-by: lachmanfrantisek <flachman@redhat.com>

Resolves partially #17

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,4 +13,7 @@
         [console_scripts]
         colin=colin.cli.colin:cli
     ''',
+    data_files=[("/usr/share/colin/", ["colin/core/config/default.json",
```
Comment (phracek) [inline setup.py]: What about to move directory `colin/core/config` to `config`? Does it make sense?

Comment (lachmanfrantisek) [reply setup.py]: Yes, it would be useful for the users, that want to see them in the source.

Comment (lachmanfrantisek) [reply setup.py]: Moved.

File: setup.py [hunk 2]
```diff
@@ -13,4 +13,7 @@
         [console_scripts]
         colin=colin.cli.colin:cli
     ''',
+    data_files=[("/usr/share/colin/", ["config/default.json",
```
Comment (TomasTomecek) [inline setup.py]: Just FYI: this won't work in virtualenv and `pip install --user .`. I'm absolutely fine with that, just sayin'.

Comment (TomasTomecek) [reply setup.py]: I vote for .local/share/colin and /usr/local/share/colin

Comment (lachmanfrantisek) [reply setup.py]: So, setting `./share/colin` in `data_files`?

And then try to read in `.local` first?

Comment (TomasTomecek) [reply setup.py]: sounds good!

**Comment by phracek** [comment]
LGTM. thx

**Comment by lachmanfrantisek** [comment]
@phracek  wait with the merge please, I'll update the target as @TomasTomecek  suggested.

**Comment by phracek** [comment]
LGTM. Let's try it.

**Pull request merged**